src/docs/01_introduction_and_goals.adoc

```diff
@@ -60,28 +59,18 @@ These quality goals are the ones which have the most importance to the major sta
 
 [role="arc42help"]
 ****
-.Contents
-Explicit overview of stakeholders of the system, i.e. all person, roles or organizations that
-
-* should know the architecture
-* have to be convinced of the architecture
-* have to work with the architecture or with code
-* need the documentation of the architecture for their work
-* have to come up with decisions about the system or its development
-
-.Motivation
-You should know all parties involved in development of the system or affected by the system.
-Otherwise, you may get nasty surprises later in the development process.
-These stakeholders determine the extent and the level of detail of your work and its results.
-
-.Form
-Table with role names, person names, and their expectations with respect to the architecture and its documentation.
+.Summary
+The following table describes the stakeholders of this project and what is expected from them at the beginning of the project, it will be updated as the project goes on if it is required. 
 ****
 
 [options="header",cols="1,2,2"]
 |===
 |Role/Name|Contact|Expectations
-| _<Role-1>_ | _<Contact-1>_ | _<Expectation-1>_
-| _<Role-2>_ | _<Contact-2>_ | _<Expectation-2>_
+| _Front-End_ | _Guillermo Fernández-Campoamor Fernández_ | _Focus on the front-end side of the app_
+| _Front-End_ | _Fernando Sáenz de Santa María Modroño_ | _Focus on the front-end side of the app_
+| _Front-End_ | _Alejandro Matas Sánchez_ | _Focus on the front-end side of the app_
+| _Back-End_ | _Andrés García Celis_ | _Focus on the back-end side of the app_
+| _Back-End_ | _Clara Miranda García_ | _Focus on the back-end side of the app_
+| _Back-End_ | _Violeta Ruiz Martínez_ | _Focus on the back-end side of the app_
+| _Back-End_ | _Miguel Rodríguez Huerta_ | _Focus on the back-end side of the app_
```

Comment:
I don't think this is the correct approach to the stakeholders. The stakeholders are every party affected by the development of the system, and here only the development team members are listed. Possible stakeholders for this project could be: 
* Development team (as a whole)
* Teachers (they are expecting a minimum quality)
* Inrupt (as we're taking part in the Inrupt challenge)
* Solid community (they need more people involved in the developing of Solid applications)
* The users of the app (they expect usability, security, etc. within the app)
* Etc.

Reply:
Giving it a second thought, I think you are right but as I see it, the development team should, at least, be divided in the different roles that we decided, I´ll change it and upload it again in a moment to see what the team thinks about it, thanks for the feedback